packages/xo-server-cloud/src/index.js

```diff
@@ -20,8 +20,10 @@ class XoServerCloud {
   }
 
   async load() {
-    const getResourceCatalog = () => this._getCatalog()
-    getResourceCatalog.description = 'Get the list of all available resources'
+    const getResourceCatalog = ({ filters } = {}) =>
+      this._getCatalog({ filters })
```

Comment:
What's the point of destructuring and then creating a new object, simply bind the method:

```js
const getResourceCatalog = this._getCatalog.bind(this)
```